misc/drone-scripts/setup.sh

```diff
@@ -13,9 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-echo "Resetting testbed"
+echo "Resetting testbed!!"
 govc snapshot.revert -vm $ESX_6_5 init
-govc snapshot.revert -vm $ESX_6_0 init
+# will revert the snapshot name to init before merging the changes to master
```

Comment:
remove this file from the PR and make sure CI esx is having `init` snapshot correctly set

Reply:
Done.